test/jdk/javax/security/auth/Subject/doAs/NestedActions.java

```diff
@@ -282,8 +280,7 @@ class Utils {
 
     static void readFile(String filename) {
         System.out.println("ReadFromFileAction: try to read " + filename);
-        AccessControlContext acc = AccessController.getContext();
-        Subject subject = Subject.getSubject(acc);
+        Subject subject = Subject.current();
```

Comment:
Couldn't we have just left these calling `Subject.getSubject` for now since these tests run with an SM enabled?